src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+
+public class DoneCommand extends Command {
+
+    /**
+     * Instantiates a new DoneCommand object.
+     * @param commandSplit user command split by spaces.
+     */
+    public DoneCommand(String[] commandSplit) {
+        super(commandSplit);
+        assert commandSplit.length > 1 && commandSplit[0].equals("done")
+                && commandSplit[1].chars().allMatch(Character::isDigit)
+                : "Must have done keyword and integer as argument";
+    }
```

Comment:
I like how you split a long line into several short ones to enhance code readability.